brainstormforce/astra
# AST-3397 - Social Icons: Update Twitter Icon

### Description

- We need to make the new Twitter / X Icon available for Users.
- JIRA - https://brainstormforce.atlassian.net/browse/AST-3397

### Screenshots

https://d.pr/i/06Bv7I

### Types of changes




- Improvement

### How has this been tested?

- Check if Twitter X icon is added as Social Icon in Header and Footer.
- Check in Backend, and Frontend Both.
- Check if Twitter X icon is added in Social Sharing.
- Check in Backend and Frontend Both.

### Checklist:
